osmdroid-android/src/main/java/org/osmdroid/views/MapView.java

```diff
@@ -667,14 +688,66 @@ public void setScrollableAreaLimit(BoundingBoxE6 boundingBox) {
      *            A lat/long bounding box to limit scrolling to, or null to remove any scrolling
      *            limitations
      */
-    public void setScrollableAreaLimitDouble(BoundingBox boundingBox) {
-        mScrollableAreaBoundingBox = boundingBox;
-    }
+	public void setScrollableAreaLimitDouble(BoundingBox boundingBox) {
+    	if (boundingBox == null) {
+    		resetScrollableAreaLimitLatitude();
+    		resetScrollableAreaLimitLongitude();
+		} else {
+			setScrollableAreaLimitLatitude(boundingBox.getActualNorth(), boundingBox.getActualSouth(), 0);
+			setScrollableAreaLimitLongitude(boundingBox.getLonWest(), boundingBox.getLonEast(), 0);
+		}
+	}
 
+	/**
+	 * @since 6.0.0
+	 */
+	public void resetScrollableAreaLimitLatitude() {
+		mScrollableAreaLimitLatitude = false;
+	}
+
+	/**
+	 * @since 6.0.0
+	 */
+	public void resetScrollableAreaLimitLongitude() {
+		mScrollableAreaLimitLongitude = false;
+	}
+
+	/**
+	 * @since 6.0.0
+	 */
+	public void setScrollableAreaLimitLatitude(final double pNorth, final double pSouth,
```

Comment:
is extra height distance in meters, pixels or degrees latitude?

Reply:
In pixels: currently including "pixel" in the parameter's name.